values.yaml

```diff
@@ -90,6 +98,25 @@ global:
   registry:
      bucket: ${PROJECT_ID}-gitlab-registry
 
+  ## https://docs.gitlab.com/charts/installation/deployment#outgoing-email
+  ## Outgoing email server settings
+  smtp:
+    enabled: "${$ENABLE_SMTP}"
+    user_name: "${SMTP_USER}"
+    password: 
+      secret: gitlab-smtp
+      key: password
```

Comment:
I think that we can manage this secret using a terraform variable. If the variable is present we create the secret and we add this fragment in the YAML. What do you think?

Reply:
I changed it as a variable.key inside is forced as password.